pages/linux/repquota.md

```diff
@@ -0,0 +1,27 @@
+# repquota
+
+> Display a summary of existing file quotas for a filesystem.
+
+- Report on all existing quotas:
+
+`repquota -all`
+
+- Report all quotas, even if there is no usage:
+
+`repquota -v`
+
+- Report on quotas for users (default), only super-user may view quotas which are not their own:
```

Comment:
So this command gives an error if not run with sudo and no users are specified? How about including sudo in its invocation?